examples/ansible/vars/default.yaml

```diff
@@ -1,6 +1,2 @@
-sumologic_otel_collector:
-  version: 0.50.0-sumo-0
-  src_config_path: config.yaml
-  memory_high: 200M
-  memory_max: 300M
-  systemd_service: false
+collector_tags: {}
```

Comment:
Content of this file is different than mentioned in https://github.com/SumoLogic/sumologic-otel-collector/pull/792/files#diff-86b02de024e35f262267d5401c2bdc11bb36047fff823fa1ae05516813fd8ebbR20-R27. 
Is it desired?

Reply:
Yes. Technically this is the set of default arguments. Maybe we shouldn't be telling users to modify it at all?

Reply:
I've changed the instructions to pass these arguments on the command line.